### ISSUE TITLE
result doesn't have Object prototype chain

### ISSUE THREAD

**Issue by vreality64** [vitalets/react-native-extended-stylesheet#101]
#### Steps to Reproduce

It's basic concept question. It could be bug or not based on situation.
There is some console warning when use React Native's `FlatList` component with created style sheet.

```tsx
const styles = {
  // I know it's not root level, However it's not bad first start step.
  container: EStylesheet.create({
    backgroundColor: 'white'
  })
}

const StyledList = () => (
  <FlatList
    contentContainerStyle={styles.container}  // it's also happen 'style' props
    // ... other FlatList props
  />
)
```

#### Expected Behavior

It shouldn't warn anything.

#### Actual Behavior

there are console warning in development mode

```
ExceptionsManager.js:84 Warning: Failed prop type: two.hasOwnProperty is not a function
    in ScrollView (at VirtualizedList.js:1028)
    in VirtualizedList (at FlatList.js:654)
    ...
```

#### Show the code

[This code](https://github.com/vitalets/react-native-extended-stylesheet/blob/master/src/sheet.js#L14) makes console warning because **`Object.create(null)` doesn't have prototype chain of `Object`**. Is this intended?

```ts
this.result = Object.create(null)
```

#### Environment

```
├─ react-native-extended-stylesheet@0.10.0
└─ react-native@0.56.0
```

#### Suggestions
`Object.create(null)` is internally okay as usage of map. However it has underlying possible error about Object prototype chain which is not easily noticed. I thinks it'd better to having prototype chain of Object in result of `create` function.

**Comment by vreality64** [comment]
@vitalets 
As you can see, [`StyleSheet.create({ ... })`](https://github.com/facebook/react-native/blob/master/Libraries/StyleSheet/StyleSheet.js#L361-L374) actually doesn't do anything on prototype. RN original stylesheet has prototype chain, because they return original object. 

Here are [code sandbox link](https://codesandbox.io/s/q3j01q82vq?autoresize=1&module=%2Fsrc%2FApp.js).

**Comment by vitalets** [comment]
@vreality64 Thank you!
Will fix it and release.
